justfile

```diff
@@ -67,7 +67,7 @@ lint:
 
 # Stratus: Lint and check code formatting
 lint-check:
-    @just _lint --check
+    @just _lint --check "-D warnings -A unused-imports"
```

Comment:
redundant/unused imports are a recent feature, not sure if I should allow it or not